docs/reference/uds-core/idam/account-lockout.md

```diff
@@ -0,0 +1,109 @@
+
+## Manually Configure Temporary Lockouts
+
+### Admin Console (Keycloak 24+)
+
+1. Sign into the Keycloak Admin Console and select your realm.
+
+2. Navigate to **Realm Settings → Security Defenses → Brute Force Detection**.
+
+3. Configure the following values:
+
+   * Brute Force Protected: `Lockout permanently after temporary lockout`
+     Enables Keycloak’s brute-force detection mechanism for the realm.
+
+   * Failure Factor: `3`
+     The number of failed login attempts within the counting window that triggers a lockout.
+
+   * Quick Login Check (ms): `1000`
+     If repeated failed attempts occur faster than this interval, Keycloak treats them as rapid-attack behavior and applies the minimum quick-login wait.
+
+   * Max Delta Time (s): `43200`
+     A rolling 12-hour window during which failed login attempts count toward the failure threshold.
+
+   * Wait Increment (s): `900`
+     The duration of a temporary lockout (15 minutes) after the failure threshold is reached.
+
+   * Minimum Quick Login Wait (s): `60`
+     The minimum delay applied when rapid successive failures occur.
+
+   * Max Failure Wait (s): `86400`
+     The maximum temporary lockout wait (24 hours). This sets the upper bound for how long Keycloak can delay a user before escalation.
+
+   * Failure Reset Time (s): `43200`
+     Controls when Keycloak resets the failure and lockout counters. Must exceed the rolling window to allow temporary lockouts to accumulate toward permanent lockout.
+
+   * Permanent Lockout: `ON`
+     Enables escalation to a permanent lockout once the configured number of temporary lockouts is exceeded.
+
+   * Max Temporary Lockouts: `3`
+     Allows one temporary lockout before escalating to a permanent lock.
+
+   * Brute Force Strategy: `MULTIPLE`
+     Defines how Keycloak handles the progression of lockout waits when repeated failures occur.
+
+4. Save and test with a non-production account.
+
+### Compliance mapping
+
+* DISA STIG (APP3510.1 / V-222432) requires:
+  * Lockout after 3 invalid attempts.
+  * A minimum 15‑minute lockout.
+  * STIG does not require the lockout to be permanent.
+
+* UDS Core provides two STIG‑compliant options, both of which satisfy these requirements:
+  1. Permanent lockout only (`MAX_TEMPORARY_LOCKOUTS = 0`)
+  1. Temporary‑then‑permanent lockout (`MAX_TEMPORARY_LOCKOUTS > 0` with proper Failure Reset Time)
+
+* NIST SP 800‑53 AC‑7 requires defining and enforcing a lockout policy. Both supported UDS Core options meet this requirement.
+
+### Compliance impact if changed if changed
+
+* `failureFactor` > 3 violates STIG expectations.
+* `maxDeltaTimeSeconds` < 900 breaks the “3 attempts in 15 minutes” requirement.
+* `waitIncrementSeconds` < 900 non‑compliant with the 15‑minute lockout minimum.
+* Increasing `maxTemporaryLockouts` changes the escalation behavior; allowed but must be documented.
+* Turning off permanent lockout is allowed, but alters the operational model and must be documented.
```

Comment:
We may just want to leave the details here to Lula mappings and compliance justifications, and note something more generic like "There may be compliance impacts when modifying lockout behavior, be aware of any environment specific requirements (NIST controls or STIG requirements) for brute-force protection."